assets/js/base/utils/address.ts

```diff
@@ -100,3 +102,62 @@ export const emptyHiddenAddressFields = <
 
 	return newAddress;
 };
+
+/*
+ * Formats a shipping address for display.
+ *
+ * @param {Object} address The address to format.
+ * @return {string | null} The formatted address or null if no address is provided.
+ */
+export const formatShippingAddress = (
+	address: ShippingAddress | BillingAddress
+): string | null => {
+	// we bail early if we don't have an address.
```

Comment:
Nitpick, sorry, but comments should begin with a capital letter.

```suggestion
	// We bail early if we don't have an address.
```